domain/secret/state/state.go

```diff
@@ -461,6 +542,16 @@ VALUES ($secretID.id)`
 // GetSecretOwner returns the owner of the secret with the given URI, returning an error satisfying
 // [secreterrors.SecretNotFound] if the secret does not exist.
 func (st State) GetSecretOwner(ctx domain.AtomicContext, uri *coresecrets.URI) (domainsecret.Owner, error) {
+	var owner domainsecret.Owner
```

Comment:
GetSecretOwner() isn't used now. It only existed to be called from the service when checking the labels.